server/server/server.js

```diff
@@ -0,0 +1,27 @@
+const prisma = require('./db/db')
+const express = require('express');
+const { PrismaClient } = require('@prisma/client');
+const prisma = new PrismaClient();
+const app = express();
+
+app.use(express.json());
+
+app.post('/users', async (req, res) => {

```

Comment:
this logic is for a different task on airtable which is the sign up api routes. Remove this logic from this PR but if you'd like make another PR for the other task since you basically have most the logic here you can go ahead and do that.